src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -48,6 +48,7 @@
     <ProjectReference Include="..\Microsoft.ML.Api\Microsoft.ML.Api.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.PCA\Microsoft.ML.PCA.csproj" />
```

Comment:
><ProjectReference Include="..\Microsoft.ML.PCA\Microsoft.ML.PCA.csproj" /> [](start = 4, length = 74)

Once you move the file to the correct location, please remember to remove this reference.